src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
140 MB/s is close to the limit of what a harddrive can deliver, and 400 MB/s is far into SSD territory. A 1 Gb/s network connection is the same as 125 MB/s, which is the theoretical maximum. Realistic transfer rates are much lower, maybe 60-70 MB/s. I can't imagine that there is an optical drive capable of reading anywhere close to this speed, which makes me wonder:
What are the sources used for this bandwidth? Network aren't fast enough, optical drives aren't fast enough - and I've never heard of anyone selling movies on SSDs...